tuplex/core/src/physical/PhysicalPlan.cc

```diff
@@ -226,14 +234,17 @@ namespace tuplex {
             outputMode = outMode;
         }
 
+        bool updateInputExceptions = hasFilter && hasInputExceptions && _context.getOptions().OPT_MERGE_EXCEPTIONS_INORDER();
```

Comment:
maybe add a comment on why this is true. I.e., a simple explanation should suffice.